CMakeLists.txt

```diff
@@ -1,111 +1,104 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.14...3.20)
```

Comment:
Ranges only make sense if you make use of new policies conditionally, which is not the case for this repository, but you'd know that if you actually read the CML files.